code/game/machinery/Sleeper.dm

```diff
@@ -134,6 +134,33 @@
 		else
 			user << "<span class='warning'>\The [src] has a beaker already.</span>"
 		return
+	else if(istype(I, /obj/item/weapon/grab))
+
+		var/mob/living/L = I:affecting
+		visible_message("[user] starts putting [I:affecting] into the [src].", 3)

```

Comment:
Before this message. We should actually check if L/I:affecting is a mob. Otherwise, you'll sit there, wait for 20 units of time, and then get told to sod off as L wasn't a mob.

```
if (!istype(L))
    user << span("notice", "This only works with humans.")
    return
```

Feedback message optional, but you get the idea.